webpay/pay/tasks.py

```diff
@@ -224,9 +225,13 @@ def start_pay(transaction_uuid, notes, user_uuid, provider_names, **kw):
 
     """
     key = notes['issuer_key']
+    source = 'marketplace' if is_marketplace(key) else 'other'
```

Comment:
meh, lets change that later if we want to, will be key for payments for firefox accounts